units/tmp.mount

```diff
@@ -23,3 +23,7 @@ What=tmpfs
 Where=/tmp
 Type=tmpfs
 Options=mode=1777,strictatime,nosuid,nodev,size=50%%,nr_inodes=1m
+
+# Make 'systemctl enable tmp.mount' work:
+[Install]
+WantedBy=local-fs.target
```

Comment:
Considering the first commit, wouldn't it be better to use `basic.target` here?

Reply:
In the last commit we disable static enablement of tmp.mount from local-fs.target, so both targets could be used with equivalent result. However, semantically tmp.mount it fits better to local-fs.target in my opinion.

Reply:
Semantically, yes, but `tmp.mount` is kinda special and might be needed earlier in the boot process. OTOH, we do have it this way in RHEL-9 and haven't got any complaints (maybe nobody enables it?), so let's leave it as it is...